app/static/src/app/components/help/DraggableDialog.vue

```diff
@@ -136,10 +212,30 @@ export default defineComponent({
     top: 10%;
     left: 20%;
 
+    .dragtarget {
+      white-space: nowrap;
+
+      .dialog-title {
+        display: inline-block;
+        overflow: hidden;
+        width: calc(100% - 48px);
+        vertical-align: bottom;
+      }
+    }
+
     .draggable-content {
       background-color: aliceblue;
       border: #ccc 1px solid;
-      height: calc(100% - 2.5rem);
+      height: calc(100% - 3.5rem);
```

Comment:
I had to make this `100% - 42px` to get this to align: 
![image](https://user-images.githubusercontent.com/1558093/218453229-4e273b9d-7032-462f-909a-2bdf6de4e613.png)

I was expecting this to be 24 perhaps (half of the 48 above)

Reply:
> I was expecting this to be 24 perhaps (half of the 48 above)

I think it's because the header is 24px and the container dialog has class `p-2` which gives 8px padding on all sides, so height of the content should be reduced by 8px + 24px + 8px...